File: attributes-common/src/main/java/io/github/solusmods/eternalcore/attributes/EternalCoreAttribute.java
package io.github.solusmods.eternalcore.attributes;

import io.github.solusmods.eternalcore.attributes.api.EternalCoreAttributes;
import io.github.solusmods.eternalcore.attributes.impl.network.EternalCoreAttributeNetwork;

public class EternalCoreAttribute {

    public static void init(){
        EternalCoreAttributeNetwork.init();
        EternalCoreAttributes.init();
        EternalCoreAttributeRegister.init();
    }
}

File: attributes-common/src/main/kotlin/io/github/solusmods/eternalcore/attributes/EternalCoreAttribute.kt
package io.github.solusmods.eternalcore.attributes

import io.github.solusmods.eternalcore.attributes.api.EternalCoreAttributes
import io.github.solusmods.eternalcore.attributes.impl.network.EternalCoreAttributeNetwork

object EternalCoreAttribute {
    @JvmStatic
    fun init() {
        EternalCoreAttributeNetwork.init()
        EternalCoreAttributes.init()
        EternalCoreAttributeRegister.init()
    }
}